packages/odyssey-react-mui/src/Breadcrumbs.tsx

```diff
@@ -98,10 +98,8 @@ const breadcrumbProviderValue: Record<
 const BreadcrumbList = ({
   children,
   homeHref,
-  maxVisibleItems: maxVisibleItemsOverride,
+  maxVisibleItems = 5,
```

Comment:
I want to name this name this number as a const outside of this function `defaultMaxVisibleItems`. It's not technically a magic number, but I wanna be clear.